privacy-configuration/tests.json

```diff
@@ -0,0 +1,630 @@
+{
+    "featuresDisabled": {
+        "name": "All features are disabled",
+        "desc": "Those tests use config1_reference.json where all features are disabled",
+        "referenceConfig": "config1_reference.json",
+        "tests": [
+            {
+                "name": "smarter encryption should be disabled",
+                "featureName": "https",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": [
+                    "safari-extension"
+                ]
+            },
+            {
+                "name": "tracker blocking should be disabled",
+                "featureName": "contentBlocking",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "tracker allowlisting should be disabled",
+                "featureName": "trackerAllowlist",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "blocking of third party tracking cookies should be disabled",
+                "featureName": "trackingCookies3p",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": [
+                    "safari-extension",
+                    "ios-browser",
+                    "android-browser",
+                    "macos-browser"
+                ]
+            },
+            {
+                "name": "reducing expiry of first party tracking cookies should be disabled",
+                "featureName": "trackingCookies1p",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": [
+                    "safari-extension",
+                    "ios-browser",
+                    "macos-browser"
+                ]
+            },
+            {
+                "name": "click to load should be disabled",
+                "featureName": "clickToPlay",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": [
+                    "safari-extension",
+                    "ios-browser",
+                    "android-browser",
+                    "macos-browser",
+                    "windows-browser"
```

Comment:
I came up with names for all of the platforms (I think they were not defined anywhere) based on the existing format (we use "ios-browser" in other set of tests)